Commands/MessageCount/pagination.go

```diff
@@ -0,0 +1,45 @@
+{{/*
+	This is the pagination code for the message leaderboard
+  
+	Recommended trigger: added reactions only
+	Trigger type: reaction
+*/}}
+{{/* dont touch */}}
+{{ $action := .Reaction.Emoji.Name }} {{/* The action being ran */}}
+{{ $validEmojis := cslice "➡" "⬅" }} {{/* Valid emojis */}}
+{{ $isValid := false }} {{/* Whether this is actually a valid embed / leaderboard embed */}}
+{{ $page := 1 }} {{/* The current page */}}
+{{ with and (eq .ReactionMessage.Author.ID 204255221017214977) .ReactionMessage.Embeds }} {{/* Checks for validity */}}
```

Comment:
nice usage of and and with (heh see wut I did there), altho usually I'm anti with cus of the change of . and I've had horrible experiences where I've been doing {{with sdict ...}} then {{if .ReactionAdded}} which took... embarrassingly long to debug, if you want an alternative: {{if $dot:=and (eq .ReactionMessage.Author.ID 204255221017214977) .ReactionMessage.Embeds}}